includes/forms/class-llms-form-field.php

```diff
@@ -784,21 +785,47 @@ protected function prepare_password_strength_meter() {
 		 */
 		$meter_settings = apply_filters( 'llms_password_strength_meter_settings', $meter_settings, $this->settings, $this );
 
-		// If scripts have been enqueued, add password strength meter script and localize with meter data.
+		// If scripts have been enqueued, add password strength meter script.
 		if ( did_action( 'wp_enqueue_scripts' ) ) {
-
-			wp_enqueue_script( 'password-strength-meter' );
-			llms()->assets->enqueue_inline(
-				'llms-pw-strength-settings',
-				'window.LLMS.PasswordStrength = window.LLMS.PasswordStrength || {};window.LLMS.PasswordStrength.get_settings = function() { return JSON.parse( \'' . wp_json_encode( $meter_settings ) . '\' ); };',
-				'footer',
-				15
+			$this->enqueue_strength_meter( $meter_settings );
+		} else { // Otherwise add it whe `wp_enqueue_scripts` is fired.
+			add_action(
+				'wp_enqueue_scripts',
+				function() use ( $meter_settings ) {
+					$this->enqueue_strength_meter( $meter_settings );
+				}
```

Comment:
This can cut down on the else condition.

```suggestion
			return $this->enqueue_strength_meter( $meter_settings );
		}
		// Otherwise add it whe `wp_enqueue_scripts` is fired.
		add_action(
			'wp_enqueue_scripts',
			function() use ( $meter_settings ) {
				$this->enqueue_strength_meter( $meter_settings );
			}
```